### ISSUE TITLE
The Training loss is 0

### ISSUE THREAD

**Issue by tingweii** [yhenon/pytorch-retinanet#221]
![圖片](https://user-images.githubusercontent.com/54320425/116672115-60600b00-a9d4-11eb-9fbc-631384df3481.png)

As this image, the classification loss is 0 and 2.30212 repeatedly. The object I want to find is very small defect, while my image size is 4608 x 3288. In this training, I resize the image to max_side = 3272 and min_side = 1824.

I don't think the classification loss would be like this. And I have no idea about this.

Could you help me to solve this problem? Thanks

**Comment by tingweii** [comment]
I've tried to adjust the alpha and beta in focal loss and the size of anchor. I think the fail of classification might be the disparity of foreground and background. In my image, the original size is 4608 x 3288, while the defect size is usually smaller than 100 x 100.

After I adjust the gamma = 6.0, alpha = 0.25 and the min_size = 2500, the max_size = 3072
the training process is like

![圖片](https://user-images.githubusercontent.com/54320425/116674628-7d4a0d80-a9d7-11eb-974d-fa76cf391a60.png)

It seems better than original. But the train batch size I can only set to 1 due to the input image size. Although I've already used RTX 3090.

**Comment by yhenon** [comment]
Your best bet is to crop your images into smaller segments, so it does not need to be resized as aggressively.

**Comment by yhenon** [comment]
Note that this repo is somewhat old and I haven't maintained it too well, so I cannot guarantee your results.

**Comment by timchiu9781** [comment]
Thank you for replying. I will try this. Although this repo is old, your code is the most simply I have see about retinanet on GitHub. Thank you.

**Comment by natantoh** [comment]
I'm having the same problem, I need to use it in large images to differentiate small objects, and all the losses are zero, because when resizing the classes are lost and all the images are similar.

**Comment by natantoh** [comment]
Correct me if I'm wrong, but 'Running Loss' should be the sum of 'Classification loss' and 'Regression Loss', which doesn't occur in the image. I'm going through the same issue as you.
How did you manage to solve this problem?
